tests/__snapshots__/pdf417-3/18/fast-180.json

```diff
@@ -14,19 +14,19 @@
       "y": 223
     },
     "topRight": {
-      "x": 375,
-      "y": 227
+      "x": -29,
```

Comment:
The [`579650a`](https://github.com/zxing-cpp/zxing-cpp/commit/579650afbd9f156d36351980c85722c498426ddc) fix seems to result in negative positions in some rotated barcodes, should the coordinates be clamped to zero? @axxel